lib/kennel/api.rb

```diff
@@ -5,6 +5,14 @@ module Kennel
   class Api
     CACHE_FILE = "tmp/cache/details"
 
+    def self.tag(api_resource, reply)
```

Comment:
can this be an instance method since all do `self.class.` ?

Reply:
Yes, or it can be a class method. It doesn't use any instance data.